test_requirements.txt

```diff
@@ -1,5 +1,6 @@
 # Core project/common application testing requirements
 
+django-celery
```

Comment:
I'm trying it locally. I had an error when exporting, so I'm rebuilding my whole environment now. If it works, I'll remove that requirement and rebuild it again to try.

The reason I haven't changed the label on this PR is that I have a couple of things I want to check with the actual tarball.